recipes/vsg/all/conanfile.py

```diff
@@ -0,0 +1,127 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import get, copy, rm, rmdir, collect_libs
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake
+import os
+
+required_conan_version = ">=1.53.0"
+
+class VsgConan(ConanFile):
+    name = "vsg"
+    description = "VulkanSceneGraph"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.vulkanscenegraph.org"
+    topics = ("vulkan", "scenegraph", "graphics", "3d")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "shader_compiler": [True, False],
+        "max_devices": [1,2,3,4],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "shader_compiler": True,
+        "max_devices" : 1,
+        "fPIC": True,  
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
+        }
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+       
+    def requirements(self):
+        self.requires("vulkan-loader/1.3.204.0", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("vulkan-loader/1.3.239.0", transitive_headers=True)
```

Reply:
Applying this suggestion as `1.3.239.0` is the most recent version we deliver in Conan 2.0 - unless there's any specific reasons to stay on 204??

Reply:
No, there is no particular reason for the .204. It simply was the version referenced by vsg 1.0.0 IIRC.